src/main/java/org/sagebionetworks/web/client/widget/entity/EvaluationSubmitterViewImpl.java

```diff
@@ -48,28 +61,57 @@
 	@UiField
 	TextBox submissionNameField;
 	@UiField
-	TextBox teamNameField;
-	@UiField
 	Heading selectedText;
 	@UiField
 	FormGroup entityFinderUI;
+	@UiField
+	Div teamComboBoxContainer;
+	@UiField
+	Radio isIndividualRadioButton;
+	@UiField
+	Radio isTeamRadioButton;
+	@UiField
+	Div contributorsPanel;
+	@UiField
+	SimplePanel registerTeamDialogContainer;
+	@UiField
+	Anchor registerMyTeamLink;
+	@UiField
+	Anchor createNewTeamLink;
+	@UiField
+	Paragraph teamIneligibleHtml;
+	@UiField
+	Paragraph noTeamsFoundUI;
+	@UiField
+	Div contributorsLoadingUI;
+	@UiField
+	Div contributorsHighlightPanel;
 	
+	private PortalGinInjector ginInjector;
+	private Select teamComboBox;
 	@Inject
-	public EvaluationSubmitterViewImpl(EvaluationSubmitterViewImplUiBinder binder, EntityFinder entityFinder, EvaluationList evaluationList) {
-		modal = (Modal)binder.createAndBindUi(this);
+	public EvaluationSubmitterViewImpl(
+			Binder binder, 
+			EntityFinder entityFinder, 
+			EvaluationList evaluationList, 
+			PortalGinInjector ginInjector
+			) {
+		widget = binder.createAndBindUi(this);
 		this.entityFinder = entityFinder;
 		this.evaluationList = evaluationList;
+		this.ginInjector = ginInjector;
 		
+		contributorsHighlightPanel.getElement().setAttribute("highlight-box-title", "Contributors");
 		evaluationListContainer.setWidget(evaluationList.asWidget());
 		initClickHandlers();
 	}
 	
 	public void initClickHandlers() {
-		okButton.addClickHandler(new ClickHandler() {
+		nextButton.addClickHandler(new ClickHandler() {
 			@Override
 			public void onClick(ClickEvent event) {
-				List<Evaluation> evaluations = evaluationList.getSelectedEvaluations();
-				if (evaluations.size() > 0) {
+				Evaluation evaluation = evaluationList.getSelectedEvaluation();
+				if (evaluation != null) {
```

Comment:
condition should not be here, should send selected evaluation back to presenter and then it should check